cmake/hsBuildInfo.inl.cmake

```diff
@@ -7,9 +7,9 @@ namespace plProduct
     constexpr uint32_t PRODUCT_BUILD_ID = @PRODUCT_BUILD_ID@;
     constexpr uint32_t PRODUCT_BUILD_TYPE = @PRODUCT_BUILD_TYPE@;
 
-    static const ST::string PRODUCT_CORE_NAME = ST_LITERAL("@PRODUCT_CORE_NAME@");
-    static const ST::string PRODUCT_SHORT_NAME = ST_LITERAL("@PRODUCT_SHORT_NAME@");
-    static const ST::string PRODUCT_LONG_NAME = ST_LITERAL("@PRODUCT_LONG_NAME@");
+    constexpr std::string_view PRODUCT_CORE_NAME = "@PRODUCT_CORE_NAME@";
+    constexpr std::string_view PRODUCT_SHORT_NAME = "@PRODUCT_SHORT_NAME@";
+    constexpr std::string_view PRODUCT_LONG_NAME = "@PRODUCT_LONG_NAME@";
     constexpr std::string_view PRODUCT_UUID = "@PRODUCT_UUID@";
 
     static const ST::string GIT_REV = ST_LITERAL("@GIT_REV@");
```

Comment:
Should we convert the other `static const ST::string` variables here to `constexpr std::string_view` as well?

Reply:
Changed all of these to constexpr to avoid any surprises in the future about initialization ordering